### DISCUSSION TITLE
How to represent an "acquisition function"?

### DISCUSSION THREAD

**Discussion by bukzor** [online-ml/river#672] [Q&A]
I'd like to implement this paper while using river:
https://arxiv.org/pdf/1807.02811.pdf

In particular, I'm don't see any good way to find a maximum of a river estimator. Should I use a second layer of optimization to search for the maxima of this unknown function? Or is there something in river that could shortcut that process?

I'm happy to use whichever estimator / optimizer makes this easiest.

**Comment by MaxHalford** [comment]
Hello there! I'm not sure Bayesian Optimization can be applied online as such. I think it would be a very difficult task to implement Bayesian Optimization in River if you don't have extensive knowledge on the topic. I personally am not very familiar with it.

To answer you: "maximum of a river estimator" doesn't mean anything to me. The rest of your questions are also a bit vague so it's a bit difficult for me to help you. You're very ambitious! 😁
